client/state/imports/constants.js

```diff
@@ -16,3 +16,5 @@ export const appStates = Object.freeze( {
 	UPLOADING: 'importer-uploading',
 	IMPORT_CLEAR: 'importer-clear',
 } );
+
+export const MAX_FILE_SIZE = 100000; // 100mb
```

Comment:
This is actually 0.1MB 
On the server side, the content-length header will be used by nginx. Nginx will return 413 directly when it sees that this content length exceeds 100MB.  The content length may not exceed 104857600 bytes but the file itself is not the only thing that is sent to the server so we need to leave a little margin when we check the file size.

I suggest we use 104857000 as the limit here. That leaves 600 bytes for headers and other meta that are sent.

Reply:
Good catch and good reasoning! https://github.com/Automattic/wp-calypso/pull/80242/commits/8cd435feda9dec8552f84c2811f6ca09c0917629